tests/fixtures/validators.yml

```diff
@@ -1,6 +1,8 @@
 amino_acid_substitution:
   should_match:
     - query: BRAF V600E
+    - query: braf v600e
+    - query: braf v600e
```

Comment:
potentially dumb q: is this just an accident, or is there a reason to put it twice?

Reply:
Accident lol